ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Loader, Grid } from 'semantic-ui-react'
+
+import { getGenesById, getRnaSeqDataByIndividual } from 'redux/selectors'
+import DataLoader from 'shared/components/DataLoader'
+import { loadRnaSeqData } from '../reducers'
+import { getRnaSeqDataLoading } from '../selectors'
+
+const RnaSeqOutliers = React.lazy(() => import('./RnaSeqOutliers'))
+const RnaSeqOutliersTable = React.lazy(() => import('./RnaSeqOutliersTable'))
+
+const OUTLIER_VOLCANO_PLOT_CONFIGS = {
+  outliers: {
+    getLocation: (({ geneId }) => geneId),
+    header: 'OUTRIDER Volcano Plot',
+  },
+  spliceOutliers: {
+    getLocation: (({ chrom, start, end }) => `${chrom}:${start}-${end}`),
```

Comment:
this should also  have 200bp upstream and downstream padding

Reply:
Okay. The latest padding is 5kb, right?

Reply:
No, the ticket says 200bp